index.js

```diff
@@ -23,6 +23,11 @@ class Modal extends Component {
       if (props.open && props.children !== this.state.children) {
          this.setState({children: props.children});
       }
+      if(props.animationDuration===0){
+        this.state.scale.setValue(1);
+      }else{
+        this.state.scale.setValue(this.props.open? 1 : 0.8);
+      }
```

Comment:
let's keep the style coherent and nice:
* space after `if` and before `(`
* spaces on both sides of `===`
* space before `{` and after `}`
* space on both sides of `?` and `:` in a ternary expression

sorry that there are no contribution guidelines, I should probably add some! :)